pcluster/pcluster_stack.py

```diff
@@ -288,7 +318,7 @@ def repl(mykey, myr, myval):
         pcluster_post_install_script.grant_read(cloud9_role)
         pcluster_config_script.grant_read(cloud9_role)
 
-        create_user = cdk.CfnParameter(self, "CreateUserAndGroups", default="false", type="String", allowed_values=['true','false'])
+        create_user = cdk.CfnParameter(self, "CreateUserAndGroups", default="false", type="String", allowed_values=['true','false'], description='Provision an additional IAM user (Researcher) account and Admin/PowerUser groups to simplify onboarding multiple users in one account.')
```

Comment:
Add to the description: 

Note: this requires IAM permissions to create a user and group.

Reply:
Done